library/alloc/src/vec/mod.rs

```diff
@@ -2792,6 +2792,50 @@ impl<T, I: SliceIndex<[T]>, A: Allocator> IndexMut<I> for Vec<T, A> {
     }
 }
 
+/// Collects an iterator into a Vec, commonly via [`Iterator::collect()`]
```

Comment:
That makes it sound like `Vec::from_iter` calls `Iterator::collect`

Reply:
_commonly [invoked] via_, not
_commonly [implemented] via_

I'll make it explicit.